multi-node/aws/README.md

```diff
@@ -6,14 +6,30 @@ View the full instructions at https://coreos.com/kubernetes/docs/latest/kubernet
 
 ### Download pre-built binary
 
+First, make sure you have imported the [CoreOS Application Signing Public Key](https://coreos.com/security/app-signing-key/).
+
+```sh
+gpg2 --keyserver pgp.mit.edu --recv-key FC8A365E
+
+#Validate the key fingerprint
+gpg2 --fingerprint FC8A365E
+# Key fingerprint = 18AD 5014 C99E F7E3 BA5F  6CE9 50BD D3E0 FC8A 365E
+```
+
+Go to the [releases](https://github.com/coreos/coreos-kubernetes/releases) and download the latest release tarball and detached signature (.sig) for your architecture.
+
 ```sh
-PLATFORM=linux-amd64
+PLATFORM=linux
 # Or
-PLATFORM=darwin-amd64
+PLATFORM=darwin
+
+#Verify GPG signature
+gpg2 --verify kube-aws-${PLATFORM}-amd64.tar.gz.sig kube-aws-${PLATFORM}-amd64.tar.gz
+
+#Extract binary
+tar zxvf kube-aws-${PLATFORM}-amd64.tar.gz
 
-wget https://coreos-kubernetes.s3.amazonaws.com/kube-aws/latest/${PLATFORM}/kube-aws
-chmod +x kube-aws
-# Add kube-aws binary to your PATH
+# Add ${PLATFORM}-amd64/kube-aws to your path
```

Comment:
No command/example for this? I suspect that it might be different based on people's set ups, but surely we can hint at something?